packages/schema/lib/functional-constraints/searchOrCreateKeys.js

```diff
@@ -3,20 +3,75 @@
 const _ = require('lodash');
 const jsonschema = require('jsonschema');
 
+const getFieldKeys = (definition, path) => {
+  const fields = _.get(definition, path, []);
+  return fields.map((field) => field.key);
```

Comment:
I forgot an `inputField` or `outputField` [can be a function](https://github.com/zapier/zapier-platform/blob/e471e2bd8fdc7733cf9ea5a9d59053746def483d/packages/schema/docs/build/schema.md#fieldorfunctionschema) to support [custom fields](https://github.com/zapier/zapier-platform/blob/e471e2bd8fdc7733cf9ea5a9d59053746def483d/packages/cli/README.md#customdynamic-fields). So we need to filter those fields that are functions and don't have a `key`.

```suggestion
  return fields.map((field) => field.key).filter((k) => k);
```